Network/Public/Include/Network/SerializationHandler.hpp

```diff
@@ -389,7 +389,7 @@ class SerializationHandler : public AbstractHandler
     }
 
     template <typename T>
-    SerializedState processIncomingMessageBody(const yas::shared_buffer& bodyBuffer, Message& message)
+    SerializedState processIncomingMessageBody(const yas::shared_buffer& bodyBuffer, Message& message)const
```

Comment:
For beauty, make a space between the const and the body of the method

Reply:
Problem resolved, do merge